tests/basic.bats

```diff
@@ -3,42 +3,43 @@
 load helpers
 
 @test "from" {
-  cid=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json --image alpine)
+  cid=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json alpine)
   buildah delete $cid
   cid=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json         alpine)
   buildah delete $cid
-  cid=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json         alpine  i-love-naming-things)
+  cid=$(buildah from alpine --pull --signature-policy ${TESTSDIR}/policy.json --name i-love-naming-things)
```

Comment:
Will this work with alpine in the first position? I see alpine in the last position in other lines below.  It's fine if it does, just a bit unusual.